extensions/notebook/package.json

```diff
@@ -506,21 +506,24 @@
           "displayName": "PySpark",
           "connectionProviderIds": [
             "MSSQL"
-          ]
+          ],
+          "blockedOnSaw": true
```

Comment:
nit: I believe that we refer to saw as "SAW" below. would be good to remain consistent.